storage/innobase/include/buf0flu.h

```diff
@@ -422,6 +435,9 @@ class FlushObserver {
 
 	/* True if the operation was interrupted. */
 	bool			m_interrupted;
+
+	/* Estimate of pages to be flushed */
+	ulint			m_estimate;
```

Comment:
Where is it initialized to zero?

Reply:
Fixed now. Added to initializer list in constructor